package.json

```
@@ -64,32 +42,32 @@
     }
   },
   "scripts": {
-    "prepare": "husky install && (forge --version || yarn foundryup) && yarn workspaces run prepare",
+    "prepare": "husky install && (forge --version || foundryup)",
     "commit": "cz",
     "prettier:check": "prettier --check 'packages/**/*.ts'",
     "prettier": "prettier --write 'packages/**/*.ts'",
     "lint": "eslint . --ext .ts",
-    "release": "yarn && lerna publish --no-private --force-publish && yarn retype:updateversion",
+    "release": "pnpm && lerna publish --no-private --force-publish && pnpm retype:updateversion",
```

Nit; I saw we use the explicit `pnpm install` now, so we should also use it here for consistency
```suggestion
    "release": "pnpm install && lerna publish --no-private --force-publish && pnpm retype:updateversion",
```